### PR TITLE
Tenant chores

### PR DESCRIPTION
**Pull request by mikedao**
* Added a tenants index method, and view. The view will only display active and approved tenants.

* Added migration to track tenant status.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -22,8 +22,10 @@
   location =              "East Timor represent#{n + 1}"
   organization =          "Bridge Builders#{n + 1}"
   Tenant.create!(
-    location:             location,
-    organization:         organization
+    location:     location,
```
Comment (mikedao) [inline db/seeds.rb]: Ignoring for clarity.

File: db/seeds.rb [hunk 2]
```diff
@@ -22,8 +22,10 @@
...
+    organization: organization,
+    active:       true,
```
Comment (mikedao) [inline db/seeds.rb]: Ignoring for clarity

File: db/seeds.rb [hunk 3]
```diff
@@ -22,8 +22,10 @@
...
+    approved:     true
```
Comment (mikedao) [inline db/seeds.rb]: Ignoring for clarity

File: app/controllers/tenants_controller.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 class TenantsController < ApplicationController
   def index
+    @tenants = Tenant.all.where(active: true).where(approved: true)
```
Comment (EmilyMB) [inline app/controllers/tenants_controller.rb]: Don't we want all the tenants actually? Isn't this only visible to the Platform Admin, who needs to be able to approve and activate/deactivate tenants?

File: app/views/tenants/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<% @tenants.each do |tenant| %>
+  <p><%= tenant.organization %></p>
+<% end %>
```
Comment (EmilyMB) [inline app/views/tenants/index.html.erb]: Maybe we don't need this? Adam already added the tenant index view, nested under the admin (admin/tenants/index.html.erb), which makes sense to me.

File: config/routes.rb [hunk 6]
```diff
@@ -4,7 +4,7 @@
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy", as: "logout"
 
-  resources :tenants, only: [:new, :create]
+  resources :tenants, only: [:new, :create, :index]
```
Comment (EmilyMB) [inline config/routes.rb]: Ditto for above comments.

File: db/migrate/20150220211042_add_columns_to_tenants.rb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+class AddColumnsToTenants < ActiveRecord::Migration
+  def change
+    add_column :tenants, :active, :boolean, default: false
+    add_column :tenants, :approved, :boolean, default: false
+  end
+end
```
Comment (EmilyMB) [inline db/migrate/20150220211042_add_columns_to_tenants.rb]: Glad the migration made it in this time :+1:

File: db/seeds.rb (hunk 1 above)
Comment (EmilyMB) [inline db/seeds.rb]: Seems reasonable.

**Comment by EmilyMB** [comment]
@mikedao Looks good. My only concern is that we seem to have duplication here. I think the tenants index should be in one place or the other (admin/tenants or tenants). Nesting it under the admin directory makes sense to me in order to be super-clear that it isn't accessible to anyone besides the Platform Admin, but I don't feel strongly either way.

File: app/controllers/tenants_controller.rb (hunk 4 above)
Comment (mikedao) [inline app/controllers/tenants_controller.rb]: This is for everyone, there is a user story where there is a tenants listing that changes based on admin changes, and this is setting it up.

File: app/views/tenants/index.html.erb (hunk 5 above)
Comment (mikedao) [inline app/views/tenants/index.html.erb]: Admin tenant view is to modify tenants listing, theres a user story that requires an open tenant listing.

**Comment by mikedao** [comment]
Like I said above, the two views are different, and when I get to the story this prepares for, the admin tenant view takes on different functionality.

File: db/seeds.rb (hunk 3 above)
Comment (Jwan622) [inline db/seeds.rb]: seems fine by me.

File: app/views/tenants/index.html.erb (hunk 5 above)
Comment (Jwan622) [inline app/views/tenants/index.html.erb]: sweet, seems good to me. We can see all active and approved tenants on some page. Sweet.

File: db/migrate/20150220211042_add_columns_to_tenants.rb (hunk 7 above)
Comment (Jwan622) [inline db/migrate/20150220211042_add_columns_to_tenants.rb]: agreed with this migration.

**Pull request merged**